src/java.xml/share/classes/jdk/xml/internal/JdkXmlFeatures.java

```diff
@@ -323,7 +323,7 @@ public boolean getFeature(int index) {
      */
     public void setFeature(int index, State state, Object value) {
         boolean temp;
-        if (Boolean.class.isAssignableFrom(value.getClass())) {
+        if (Boolean.class.isInstance(value)) {
```

Comment:
Could this be `value instanceof Boolean`?

Reply:
Looks like instanceof is preferable in this case. Replaced.